studio/schemas/documents/lesson.js

```diff
@@ -34,6 +34,19 @@ export default {
       name: 'awsFilename',
       type: 'url',
     },
+    {
+      name: 'resource',
+      description:
+        'Attach a resource to this lesson (Video, Audio, Text, etc.)',
+      title: 'Resource',
+      type: 'reference',
+      to: [
+        {
+          type: 'videoResource',
+          title: 'Video',
+        },
+      ],
+    },
```

Comment:
Lessons can contain a _primitive_ resource, for now that is just the `VideoResource`.